### PR TITLE
8340407: Open source a few more Component related tests

### PR DESCRIPTION
**Pull request by azuev-java**
Clean up and open source four tests.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340407](https://bugs.openjdk.org/browse/JDK-8340407): Open source a few more Component related tests (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21234/head:pull/21234` \
`$ git checkout pull/21234`

Update a local copy of the PR: \
`$ git checkout pull/21234` \
`$ git pull https://git.openjdk.org/jdk.git pull/21234/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21234`

View PR using the GUI difftool: \
`$ git pr show -t 21234`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21234.diff">https://git.openjdk.org/jdk/pull/21234.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21234#issuecomment-2380010382)

### CODE REVIEW

File: test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+*/
+
+import java.awt.Button;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.lang.reflect.InvocationTargetException;
+
+public class GetSizesTest extends Frame {
+    Button b;
```
Comment (meiftikaralam) [inline test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java:40]: nit: its better to use full name for the variable for better readability

File: test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+        Dimension dimPref = getPreferredSize();
+        dimPref.setSize(101, 101);
+        if (getPreferredSize().equals(new Dimension(101, 101))) {
+            throw new RuntimeException("Test Failed for: " + dimPref);
+        }
+        System.out.println("getPreferredSize() Passed.");
+
+        Dimension dimMin = getMinimumSize();
+        dimMin.setSize(101, 101);
```
Comment (meiftikaralam) [inline test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java:76]: why `101` value is used across? can we use a constant with a proper variable name to make the code more readable.

File: test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        }
+        System.out.println("getMinimumSize() Passed.");
+
+        dimMax = b.getMaximumSize();
+        dimMax.setSize(33, 33);
+        if (b.getMaximumSize().equals(new Dimension(33, 33))) {
+            throw new RuntimeException("Test Failed for: " + dimMax);
+        }
+        System.out.println("getMaximumSize() Passed.");
+        System.out.println("GetSizesTest Succeeded.");
```
Comment (meiftikaralam) [inline test/jdk/java/awt/Component/MinMaxSizeDefensive/GetSizesTest.java:111]: we can make the code more modular by having private methods for different tests

in addition to this we can reduce number of S.O.P prints.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**